### PR TITLE
feat: new rules and fixes

### LINKED ISSUE #3742: Exemption on proc_creation_win_creation_mavinject_process_injection.yml
mavinject process should be excluded from devices running App-V as it is normal behavior.  From [LOLBAS Project](https://lolbas-project.github.io/lolbas/Binaries/Mavinject/)

> IOC: mavinject.exe should not run unless APP-v is in use on the workstation

The logic can also be found in the rule on [SentinelOne](https://github.com/keyboardcrunch/SentinelOne-ATTACK-Queries/blob/master/Tactics/DefenseEvasion.md)

Recommend adding ParentImage Not EndsWith '/AppVClient.exe' to the rule to reduce potential false positives.

### PR DESCRIPTION
**Pull request by nasbench**
This PR covers the following:

- Fix #3742
- Added proposed changes from #2712 and marked it as closed (due to inactivity) 
- Add new rules related to scheduled tasks
- Added/Updated metadata and logic of some rules (see `Files changed` section for full details)

### CODE REVIEW

File: rules/windows/builtin/taskscheduler/win_task_scheduler_susp_task_locations.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+title: Suspicious Scheduled Tasks Locations
+id: 424273ea-7cf8-43a6-b712-375f925e481f
+status: test
```
Comment (frack113) [inline rules/windows/builtin/taskscheduler/win_task_scheduler_susp_task_locations.yml:3]: status: experimental

File: rules/windows/process_creation/proc_creation_win_lolbin_wlrmdr.yml [hunk 2]
```diff
@@ -4,29 +4,35 @@ status: experimental
...
             - '-a '
             - '-u '
+    selection_parent: # This selection is looking for processes spawned from wlrmdr using the "-u" flag
+        ParentImage|endswith: '\wlrmdr.exe'
     filter:
         ParentImage: 'C:\Windows\System32\winlogon.exe'
     filter_null:
         ParentImage: '-'
-    condition: selection and not 1 of filter*
+    condition: selection_parent or (all of selection_child_* and not 1 of filter*)
```
Comment (frack113) [inline rules/windows/process_creation/proc_creation_win_lolbin_wlrmdr.yml:35]: condition: (selection_parent or all of selection_child_* ) and not 1 of filter*

File: rules/windows/process_creation/proc_creation_win_susp_conhost.yml [hunk 3]
```diff
@@ -46,8 +46,9 @@ detection:
         Image: 'C:\Windows\System32\conhost.exe'
     filter_image_conhost2:
         ParentCommandLine:
-            - '\??\C:\WINDOWS\system32\conhost.exe 0x4'
-            - '\??\C:\WINDOWS\system32\conhost.exe 0xffffffff -ForceV1'
+            # The "?" needs to be escaped as they considered special characters in SIGMA (https://github.com/SigmaHQ/sigma-specification)
```
Comment (frack113) [inline rules/windows/process_creation/proc_creation_win_susp_conhost.yml:49]: miss the modified field

File: rules/windows/process_creation/proc_creation_win_susp_conhost.yml [hunk 4]
```diff
@@ -46,8 +46,9 @@ detection:
...
+            - '\\?\?\C:\WINDOWS\system32\conhost.exe 0x4'
+            - '\\?\?\C:\WINDOWS\system32\conhost.exe 0xffffffff -ForceV1'
```
Comment (phantinuss) [inline rules/windows/process_creation/proc_creation_win_susp_conhost.yml:51]:
```suggestion
            - '\\\?\?\C:\WINDOWS\system32\conhost.exe 0x4'
            - '\\\?\?\C:\WINDOWS\system32\conhost.exe 0xffffffff -ForceV1'
```
the 1st `\` escapes the 2nd one. So we need 3 `\`s to get a literal `\?`.

**Review by phantinuss — APPROVED** [review]

**Pull request merged**